packages/plugins/RSS3/src/SNSAdaptor/components/ReversedAddress.tsx

```diff
@@ -0,0 +1,26 @@
+import { memo } from 'react'
+import { isSameAddress, NetworkPluginID } from '@masknet/web3-shared-base'
+import { useReverseAddress, useWeb3State } from '@masknet/plugin-infra/web3'
+import { ZERO_ADDRESS } from '@masknet/web3-shared-evm'
+
+interface ReverseAddressProps {
+    address?: string
+    pluginId?: NetworkPluginID
+    domainSize?: number
+    size?: number
+    fontSize?: string
+    fontWeight?: number
+}
+
+export const ReversedAddress = memo<ReverseAddressProps>(
```

Comment:
Again why is this component needed even though we have a similar one in `@masknet/shared`?